.devcontainer/scripts/configure-proxies.sh

```diff
@@ -13,14 +13,9 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-if [ "${PROXY}" = ".Proxy" ]; then
-    USE_PROXIES="true"
-    HTTP_PROXY="${PROXY_HOST}:${PROXY_PORT}"
-    HTTPS_PROXY=$HTTP_PROXY
-    FTP_PROXY=$HTTP_PROXY
-    ALL_PROXY=$HTTP_PROXY
-    NO_PROXY="localhost,127.0.0.1,0.0.0.0,10.0.0.0/8,192.168.122.0/24,cattle-system.svc,.svc,.cluster.local"
-fi
```

Comment:
I would like to leave this for now as it is because we have to sync the proxy settings anyhow later